articles/container-service/container-service-connect.md

```diff
@@ -27,19 +27,25 @@ described below.
 
 ## Connecting to a Kubernetes cluster.
 To connect to a Kubernetes cluster, you need to have the `kubectl` command line tool installed.  The easiest way to install this
-tool is to use the Azure 2.0 `az` command line tool.
+tool is to use the Azure 2.0 [`az` command line tool.](https://github.com/azure/azure-cli#install-with-pip)
 
 ```console
-az acs kubernetes install cli [--install-location=/some/directory]
+# Linux or OS X
+az acs kubernetes install-cli [--install-location=/some/directory/kubectl]
+
+# Windows
+az acs kubernetes install-cli [--install-location=C:\some\directory\kubectl.exe]
 ```
 
 Alternately, you can download the client directly from the [releases page](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG.md#downloads-for-v146)
 
+### Installing kubectl credentials
+
 Once you have `kubectl` installed, you need to copy the cluster credentials to your machine.  The easiest way to do
 this is again the `az` command line tool:
 
 ```console
-az acs kubernetes get-credentials --dns-prefix=<some-prefix> --location=<some-location>
+az acs kubernetes get-credentials --resource-group=<cluster-resource-group> --name=<cluster-name>
```

Comment:
No, it's not necessary here, when you use resource group / name, you pull the cluster info, and it grabs the correct user from the cluster info.